tests/telemetry/test_telemetry.py

```diff
@@ -266,12 +266,11 @@ def process_trace(self, trace):
     ]["message"]
 
 
+@pytest.mark.skip(reason="We don't have a way to capture unhandled errors in bootstrap before telemetry is loaded")
```

Comment:
I think this is fine to merge. It's worth noting that there's probably a straightforward way to keep track of such errors before starting the telemetry writer worker, sending the associated telemetry events when it does eventually start.

Reply:
yes, but, maybe?

if the errors occur and cause the application to exit before the telemetry writer is started

we probably could have an atexit hook that loads and flushes the telemetry events even if the writer hasn't been started 🤔